### DISCUSSION TITLE
Hera "community maintained templates"

### DISCUSSION THREAD

**Discussion by flaviuvadan** [argoproj-labs/hera#677] [Ideas]
Maybe there's space for Hera to offer community maintained templates. These templates could live in the Hera repository, be offered as `WorkflowTemplate` object, which can be taken, created, and used instantly by users

**Comment by elliotgunton** [comment]
With the [CLI](https://github.com/argoproj-labs/hera/issues/670) `generate` features we could kick start this idea with WorkflowTemplate stubs or full Python from yaml of existing sources! 🚀

**Reply by flaviuvadan** [reply]
That would be really amazing 🚀
